boltdesignsystem/bolt
# Fix: text-contrast min ratio and icon background border

## Jira

N/A

## Summary

1. Fixes a bug where the `bolt-text-contrast()` function's min ratio is set 3.1 instead of 3. 
2. Fixes a bug where the icon component has a border around the colored background.

## How to test

Run the branch locally and test the `bolt-text-contrast()` function, try 2 pairs of colors, one pair with 3:1 contrast ratio, and another with 2.9:1 contrast ratio. For example:

```
.test {
  color: bolt-text-contrast(#ab81d5);
  background-color: #ab81d5;
}
```

Then browse through the icon docs. Make sure there's no border unless the background color is set to white.
